MonoGame.Framework/Platform/Graphics/Effect/Resources/Structures.fxh

```diff
@@ -80,7 +80,7 @@ struct VSInputNmTxWeights
     float4 Position : POSITION0;
     float3 Normal   : NORMAL0;
     float2 TexCoord : TEXCOORD0;
-    uint4  Indices  : BLENDINDICES0;
```

Comment:
This works around an issue DXC/Glsl where unsigned int 4 didn't compile.  This shouldn't matter as you never have huge numbers for blend indices.